templates/rbac.yaml

```diff
@@ -68,17 +26,11 @@ rules:
       - "" # indicates the core API group
       - "apps"
     resources:
-      # Every environment deployment has a service.
-      - services
       - pods
       # Required for creating registry pull secrets.
       - secrets
-      # Required for the code-server volume and the environments' home volumes.
+      # Required for the environments' home volumes.
       - persistentvolumeclaims
-      # Required for legacy reasons.
-      - statefulsets
```

Comment:
We have 2 kubemigrations that read statefulsets, so we need to delete those first